js/web/lib/wasm/jsep/webgpu/ops/where.ts

```diff
@@ -27,7 +27,7 @@ const createWhereOpProgramShader =
           const expressionA = `a_data[index_a${x}][component_a${x}]`;
           const expressionB = `b_data[index_b${x}][component_b${x}]`;
           // eslint-disable-next-line no-bitwise
-          const expressionC = `bool(c_data[index_c${x}] & ${0xff000000 >>> ((3 - x) * 8)}u)`;
+          const expressionC = `bool(c_data[index_c${x}] & (0xffu << (component_c${x} * 8)))`;
```

Comment:
just curious: is there a reason to use Lshift to replace Rshift?

Reply:
Computing the left-shift takes one less arithmetic operation and relatively easy to understand and left-shift always shifts-in 0s unlike right shift, where one needs to be careful to use unsigned-right-shift instead of signed-right-shift.